### DISCUSSION TITLE
[SP-API] Authorization Private App

### DISCUSSION THREAD

**Discussion by christian-vr** [amzn/selling-partner-api-models#3816] [General]
Hi, I am interested in developing an app to be used only by some specific Seller Accounts (not in my organization). I would prefer to have it as private instead of public since is not going to be of use by any other Seller Accounts other than those.

I saw the Self Authorization flow but it looks like I could do it in my account because I can see the app and Authorize it there but these other Sellers won't see in their accounts.

Is this possible? How would these Sellers be able to authorize the app or get a token for me to use in my app?

Thanks!

**Comment by mint18** [comment]
Any ideas or solutions from your side? It seems we have a similar problem.

**Reply by christian-vr** [reply]
Hey, I got in contact with support - it has to be a public app.
